File: main/goblin.py
from random import randint
from time import time
def domath(number):
    d = pow(number, 20)
    i = pow(d, 20)
def main():
    r = time()
    i = 0
    while(i < 50000000):
        n = randint(1, 10000000)
        domath(n)
        i += 1
        print(i)
    ert = time()
    est = ert - r
    print(est, 's')
if __name__ == "__main__":
    main()

File: main/goblin.go
package main
import(
	"fmt"
	"math"
	"math/rand"
	"time"
)
func domath(number float64) {
	var d = math.Pow(number, 20)
	var id = math.Pow(d, 20)
	id++
}
func main(){
	rand.Seed(time.Now().UnixNano())
	fmt.Println(rand.Intn(100))
	for i:=0; i<50000000; i++{
		var n int = (rand.Intn(10000000))
		f := float64(n)
		domath(f)
	}

}
